roles/container-engine/docker/templates/docker-options.conf.j2

```diff
@@ -1,11 +1,11 @@
 [Service]
 Environment="DOCKER_OPTS={{ docker_options|default('') }} --iptables={{ docker_iptables_enabled | default('false') }} \
 --exec-opt native.cgroupdriver={{ docker_cgroup_driver }} \
-{% for i in docker_insecure_registries %}--insecure-registry={{ i }} {% endfor %} \
-{% for i in docker_registry_mirrors %}--registry-mirror={{ i }} {% endfor %} \
+{% for i in docker_insecure_registries %}--insecure-registry={{ i }} \{% endfor %}
```

Comment:
Doesn't this leave empty lines instead of just `\`?

Reply:
Doesn't looks like it
```
[root@instance-1 docker.service.d]# cat docker-options.conf
[Service]
Environment="DOCKER_OPTS= --iptables=false \
--exec-opt native.cgroupdriver=systemd \
--registry-mirror=https://mirror.gcr.io \--data-root=/var/lib/docker \
--log-opt max-size=50m --log-opt max-file=5 \"
```

Reply:
That `--registry-mirror=https://mirror.gcr.io \--data-root=/var/lib/docker \` still looks messy and is something unexpected for sure.

Reply:
Indeed 👍